sass/_typography.scss

```diff
@@ -8,182 +8,184 @@ select,
 textarea {
 	color: $text-color-base;
 	font-family: $font-stack-content;
-    @include rem(font-size, $font-size-normal);
+	@include rem(font-size, $font-size-normal);
 	line-height: 1.5;
 }
 
-h1, 
-h2, 
-h3, 
-h4, 
-h5, 
+h1,
+h2,
+h3,
+h4,
+h5,
 h6 {
 	clear: both;
 }
 
-h1, 
-h2, 
-h3, 
+h1,
+h2,
+h3,
 h4 {
 	color: $text-color-heading;
-    @include rem(font-size, 24px);
+	@include rem(font-size, 24px);
 	margin: 30px 0;
 }
 
 h1 {
-    font-family: $font-stack-heading;
-    @include rem(font-size, 58px);
+	font-family: $font-stack-heading;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets